docs/development/setup_development.rst

```diff
@@ -12,6 +12,8 @@ specific type of development task. We use Vagrant, VirtualBox, and
 Docker and our Ansible playbooks can provision these environments on
 either virtual machines or physical hardware.
 
+SecureDrop uses Python2. We are migrating to Python3. Some parts of the application are built with Python3. 
```

Comment:
There is no migration effort to python3 at the moment. A standalone GUI tool was recently added and uses python3 but this is the only part that is python3.